custom_otello_linter/helpers/find_make_screenshot_calls.py

```diff
@@ -0,0 +1,24 @@
+import ast
+
+from custom_otello_linter.helpers.get_full_func_name import get_full_func_name
+
+
+def find_make_screenshot_calls(step_body):
+    """
+    Проверяет наличие вызова функции make_screenshot_for_comparison в тесте.
+    Возвращает список узлов AST, соответствующих этим вызовам
```

Comment:
```suggestion
def get_make_screenshot_calls(step: FuncType) -> List[ast.Call]:
    """
    Проверяет наличие вызовов функции make_screenshot_for_comparison в шаге.
    Возвращает список узлов AST, соответствующих этим вызовам
```
1) Лучше `get_`, потому что функция не только находит, но и возвращает значения
2) Можно упростить использование функции в чекерах, если просто передавать шаг, а не свойство `body`. А чтобы гарантировать, что в передаваемом объекте будет свойство `body`, укажем тип `FuncType` для параметра. Так мы сможем меньше думать в местах, где вызывается этот хелпер.